Misc/NEWS.d/next/Library/2025-02-25-03-53-00.gh-issue-130461.asr2dg.rst

```diff
@@ -0,0 +1,3 @@
+* Removed ``.. index::`` directives from uuid module documentation.
+* These directives previously created entries in the general index for functions ``getnode``, ``uuid1``, ``uuid3``, ``uuid4``, ``uuid5``, and ``uuid8``.
+* This change affects the intersphinx inventory. (GH-130461)
```

Comment:
```suggestion
Remove ``.. index::`` directives from the :mod:`uuid` module documentation. These directives
previously created entries in the general index for :func:`~uuid.getnode` as well as the
:func:`~uuid.uuid1`, :func:`~uuid.uuid3`, :func:`~uuid.uuid4`, :func:`~uuid.uuid5`, and
:func:`~uuid.uuid8` constructor functions.
```

I think it's fine to just mention their removal. It's implicit that the intersphinx inventory would have been modified.

Reply:
Thanks @picnixz for the suggestion ❤️  , I've committed them into this branch.